test/unit/server_test.js

```diff
@@ -255,4 +280,148 @@ describe("server.js tests", function () {
       expect(typeof json.test).to.be.equal("function");
     });
   });
+
+  describe("Smoke tests for the rest of the builders", function() {
+    describe("AccountCallBuilder", function() {
+      beforeEach(function (done) {
+        // instruct the dev server to except the correct request
+        return this.setFixtures({
+          request: "/accounts/GSDEF?limit=4",
+          response: {status: 200, body: FAKE_COLLECTION_RESPONSE}
+        }).then(function () { done() });
+      });
+
+      it("requests the correct endpoint", function (done) {
+        server.accounts()
+          .address("GSDEF")
+          .limit("4")
+          .call()
+          .then(function () {
+            done();
```

Comment:
Could you explain more on how we should check the response?  I'm not sure what you mean.